air_hockey:
  puck_tracker:
    max_prediction_steps:
      120
    frequency:
      120
    friction_drag:
      0.13087578868418712
    friction_sliding:
      1.5363436894411218e-06
    restitution_table:
      0.79
    restitution_mallet:
      0.8
    observation_variance_position:
      2.5e-7
    observation_variance_angular:
      2.5e-7
    dynamic_variance_position:
      1.0e-6
    dynamic_variance_velocity:
      1.0e-4
    dynamic_variance_angular_position:
      1.0e-6
    dynamic_variance_angular_velocity:
      1.0e-4